runtime/compiler/env/j9method.cpp

```diff
@@ -5299,6 +5331,15 @@ TR_ResolvedJ9Method::startAddressForJITInternalNativeMethod()
    return startAddressForJittedMethod();
    }
 
+#if defined(JITSERVER_SUPPORT)
+TR_PersistentJittedBodyInfo *
```

Comment:
Let's eliminate `#if defined(JITSERVER_SUPPORT)` for this one as it is needed in a few places in the optimizer/codegen.
In a follow-up PR (for master branch) use getExistingJittedBodyInfo() in all those places.